tests/python_client/testcases/test_collection.py

```diff
@@ -2862,7 +2863,16 @@ def test_load_replica_partitions(self):
                            check_task=CheckTasks.check_query_results,
                            check_items={'exp_res': df_2.iloc[:1, :1].to_dict('records')})
 
-        error = {ct.err_code: 1, ct.err_msg: f"not loaded into memory"}
+        error = {ct.err_code: 65538, ct.err_msg: "failed to query: attempt #0: failed to search/query delegator "
```

Comment:
why we assert such a long and specific error msg? the partition id shall be changed next time

Reply:
I will modify the message and extract keywords.